textproc/lout/Portfile

```diff
@@ -1,4 +1,5 @@
 PortSystem 1.0
+PortGroup  compiler_blacklist_versions 1.0
```

Comment:
You're not using this portgroup. But you should use it, to blacklist specific versions of Xcode clang, such as `compiler.blacklist-append {clang >= 802 < 900}` (replace these version numbers with the actual version numbers that are affected by this bug). This should be done outside of any platform-version-specific block, assuming it would affect those versions of Xcode clang on any systems where they work.